### DISCUSSION TITLE
How does the unique visitor count work?

### DISCUSSION THREAD

**Discussion by shreydd** [plausible/analytics#2771] [Anything Else]
Many services I use have a bit of a lose definition with their description of `unique visitor` count. 

For example, if I visit my link through an incognito tab on the same logged in device, it registers it as a unique viewer count. I'm assuming plausible doesn't count that as a view? If so then how does it work? Would love to know more about it. 

Cheers.

**Comment by metmarkosaric** [comment]
thanks for reaching out @shreydd! here's a good link that explains how our method for tracking unique visitors works in detail: https://plausible.io/data-policy

**Reply by shreydd** [reply]
It's such a cool solution you folks have cracked, Thank you Marko!
